cvat/apps/iam/apps.py

```diff
@@ -1,8 +1,24 @@
+from distutils.util import strtobool
+import os
+from pathlib import Path
+import tarfile
 from django.apps import AppConfig
+from django.conf import settings
 
 class IAMConfig(AppConfig):
     name = 'cvat.apps.iam'
 
     def ready(self):
         from .signals import register_signals
         register_signals(self)
+
+        if strtobool(os.environ.get("IAM_BUNDLE_RULES", '0')):
+            bundle_path = Path(settings.IAM_RULE_BUNDLE_PATH)
```

Comment:
Please use the same pattern for names. For now we have IAM_RULE_BUNDLE, IAM_BUNDLE_RULES.

I will recommend something like IAM_OPA_BUNDLE